ChiModules/LinearBoltzmannSolver/lbs_03d_wgdsa.cc

```diff
@@ -110,7 +110,7 @@ void LinearBoltzmann::Solver::AssembleWGDSADeltaPhiVector(LBSGroupset& groupset,
   int gss = groupset.groups.size();
 
   delta_phi_local.clear();
-  delta_phi_local.resize(local_dof_count*gss,0.0);
+  delta_phi_local.resize(local_node_count * gss, 0.0);
```

Comment:
Just an interesting observation that resize followed by fill might be faster than clear and resize.

Reply:
What would you like to see here?

Reply:
No fix necessary.  Just an observation.